### DISCUSSION TITLE
Not getting email comformation emails

### DISCUSSION THREAD

**Discussion by RedWedgeX** [community/community#147707] [GitHub Education]
### Hi everyone,

I've been trying to add my school email to Github so I can sign up for the student developer pack, but I'm just not receiving the confirmation emails. I've tried resending them multiple times, still nothing. I confirmed the email was correct, and that it can receive emails by sending from a few other addresses outside the school.

What's even weirder is when I go to DELETE the unconfirmed address off my account, I get an email confirming that. 
![image](https://github.com/user-attachments/assets/423cdff1-da24-44d3-ba8e-613c5508a337)

Thoughts? Is there any other way to get myself verified?

**Comment by davevad93** [comment]
Hi @RedWedgeX ,

That's strange, since you can receive emails from other addresses outside your school domain, even from GitHub, although GitHub Education's is `edu-noreply@github.com`, have you tried setting your academic email as the primary email address in your account and also checking the spam folder?

**Reply by davevad93** [reply]
Ah sorry, I see your problem is that you can't verify your academic email on GitHub, my bad I thought you meant the confirmation email from GitHub Education about your application.

**Reply by davevad93** [reply]
That's strange indeed, because the email sent by GitHub to verify an email address is from `noreply@github.com` and you received an email from them about removing your academic email address from your account, so it shouldn't be a problem with your email. You can try contacting support by opening a ticket [here](https://support.github.com/contact/account), select `Other` and explain your problem.

**Reply by RedWedgeX** [reply]
Oof, yeah, good call. I'll send up a ticket. Thanks for the assist!

**Reply by davevad93** [reply]
You're welcome @RedWedgeX 👍
